APIManager.js

```diff
@@ -0,0 +1,165 @@
+//This is the class that will manage all your APIs
+
+class APIManager
+{
+    constructor()
+    {
+        this.data = {};
+    }
+
+    loadData()
+    {
+        this.data.users = new UsersAPI().loadData();
+        this.data.quotes = new QuotesAPI().loadData();
+        this.data.bacons = new BaconsAPI().loadData();
+        this.data.pokemons = new PokemonsAPI().loadData();
+        return this.data;
+    }
+
+    // set data(data)
+    // {
+    //     this._data = data;
+    // }
+
+    // get data()
+    // {
+    //     return this._data;
+    // }
+}
+
+class UsersAPI extends APIManager
+{
+    constructor()
+    {
+        super();
+    }
+
+    loadData()
+    {
+        const _data = [];
```

Comment:
there is no need for `_data` to be an array, it could be just the data that you need from the external API